watertap/unit_models/electrodialysis_1D.py

```diff
@@ -658,7 +658,7 @@ def build(self):
             self.flowsheet().time,
             self.diluate.length_domain,
             initialize=0.9,
-            bounds=(0, 1),
+            bounds=(0, 1 + 1e-10),
```

Comment:
What happened without this?

Reply:
At least one test fails, and the solves look uglier..
https://github.com/bknueven/watertap/actions/runs/6423080058/job/17440867424?pr=11

Reply:
If it were possible to eliminate these bounds that would probably be better.